change/@ni-nimble-blazor-1e92e494-e932-4668-8797-d5a971330f7a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
The build is showing diffs in a bunch of anchor button stories. I'm guessing you just need to update from main but if that doesn't work we should try to figure out what's up.

Reply:
@jattasNI it doesn't anymore correct?

Reply:
Correct!